posthog-android/src/main/java/com/posthog/android/replay/PostHogSessionReplayConfig.kt

```diff
@@ -48,4 +48,11 @@ public class PostHogSessionReplayConfig
          */
         @PostHogExperimental
         public var debouncerDelayMs: Long = 500,
+        /**
+         * Define the minimum duration for sessions to be recorded.
+         * This is useful if you want to exclude sessions that are too short to be useful.
+         * Defaults to 0ms
+         */
+        @PostHogExperimental
+        public var minSessionDurationMs: Long = 0,
```

Comment:
```suggestion
        public var minSessionDurationMs: Long? = null,
```

I think nullable makes more sense than 0, which means no limits